docs/docs/llms/llm-setup.mdx

```diff
@@ -70,14 +79,19 @@ to be configured:
   OpenAI Service.
 - `openai.api_base`: This should be the URL for your Azure OpenAI instance. An
   example might look like this: "https://docs-test-001.openai.azure.com/".
-- `openai.api_version`: This should be the version of Azure OpenAI you are
-  using, such as "2023-05-15".
+
 
 To configure these parameters, follow these steps:
 
-1. [Instructions to set `openai.api_type`]
-2. [Instructions to set `openai.api_base`]
-3. [Instructions to set `openai.api_version`]
+1. To configure the `openai.api_type` as an environment variable:
+  ```shell
+  export OPENAI_API_TYPE="azure"
+  ```
+2. To configure the `openai.api_base` as an environment variable:
```

Comment:
same as above, can you describe what "api_base" is and why someone needs to configure it?

Reply:
do we want to give instructions on how to set `api_version`?

Reply:
`api_version` is defined based on `api_type`
https://github.com/openai/openai-python/blob/main/openai/__init__.py#L51

If we want to make `api_version` configurable, we need to change the approach of configuring from using environment variables to using policies configuration